processor/transformprocessor/README.md

```diff
@@ -415,6 +416,50 @@ statements:
 
 To aggregate only using a specified set of attributes, you can use `keep_matching_keys`.
 
+### aggregate_on_attribute_value
+
+`aggregate_on_attribute_value(function, attribute, values, newValue)`
+
+The `aggregate_on_attribute_value` function aggregates all datapoints in the metric containing the attribute `attribute` with one of the values present in the `values` parameter. `function` is a case-sensitive string that represents the aggregation function.
+
+Firstly, `attribute` values with one of the values present in `values` are substituted by `newValue` for all datapoints. Afterwards all datapoints are aggregated depending on the attributes.
```

Comment:
```suggestion
The `aggregate_on_attribute_value` function aggregates all datapoints in the metric containing the attribute `attribute` with one of the values present in the `values` into a single datapoint where the attribute has the value `newValue`. `function` is a case-sensitive string that represents the aggregation function.
```

I think we could make this a bit more concise by just explaining how the function uses the parameters to operate on the metric.